packages/page-accounts/src/Sidebar/AddressSection.tsx

```diff
@@ -46,15 +61,36 @@ function AddressSection ({ accountIndex, defaultValue, editingName, flags, onCha
           value={value}
           withSidebar={false}
         />
+       
         <div className='ui--AddressMenu-addr'>
           {value}
         </div>
+        
         {accountIndex && (
           <div className='ui--AddressMenu-index'>
             <label>{t<string>('index')}:</label> {accountIndex}
           </div>
         )}
       </div>
+
+      <div className='ui--AddressSection__CopyColumn'>
+        <div className='ui--AddressMenu-copyaddr'>         
+          <CopyToClipboard text={value}
+          onCopy={() => setCountCopied(countCopied + 1)}
+          >
+            <span >
+              <Button.Group> 
+                <Button  
+                    icon={countCopied ? 'check' : 'copy'}   
+                    label={t<string>('Copy')} 
+                    onClick={() => setCountCopied(countCopied + 1) }
+                    onMouseLeave={() => setCountCopied(0) } 
```

Comment:
As above, use `useIncrement` if you wish to increment or `useToggle` if you want a true/false (which is probably better suited here)